Zilon.Core/Zilon.Core/Tactics/Sector.cs

```diff
@@ -221,7 +220,7 @@ private void UpdateEquipments()
             }
         }
 
-        private void PropContainerManager_Added(object sender, ManagerItemsChangedEventArgs<IPropContainer> e)
+        private void StaticObjectManager_Added(object sender, ManagerItemsChangedEventArgs<IStaticObject> e)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unused method parameter 'sender'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5353580)